Wikipedia/Code/ActivityTabViewController.swift

```diff
@@ -0,0 +1,215 @@
+import WMFData
+import CocoaLumberjackSwift
+import WMFComponents
+import WMF
+import Combine
+
+final class WMFActivityTabHostingController: WMFComponentHostingController<WMFActivityTabView> {}
+
+@objc final class WMFActivityTabViewController: WMFCanvasViewController, WMFNavigationBarConfiguring, WMFNavigationBarHiding {
+    
+    var topSafeAreaOverlayView: UIView?
+    
+    var topSafeAreaOverlayHeightConstraint: NSLayoutConstraint?
```

Comment:
Can you try removing these? I'm not certain they are needed (WMFHistoryViewController doesn't have it).